packages/experimental/Button/src/FAB/FAB.styling.ts

```diff
@@ -31,11 +31,11 @@ export const stylingSettings: UseStylingOptions<ButtonCorePropsWithInnerRef, But
         style: {
           color: tokens.color,
           ...getTextMarginAdjustment(),
-          ...(tokens.spacingIconContent && { marginLeft: tokens.spacingIconContent }),
+          ...(tokens.spacingIconContentBefore && { marginLeft: tokens.spacingIconContentBefore }),
```

Comment:
Should be marginStart to account for RTL?

Reply:
Yeah, I should switch to marginStart/end here, thanks for the catch. I think in reality, RTL will flip marginLeft/Right in RN so that marginLeft will be applied to the right in RTL, according to https://github.com/microsoft/react-native-windows/issues/4645#issuecomment-616890884.

Reply:
Oh.... that sounds wrong to me. I thought the whole point of marginStart/End was so that marginLeft is __always_ left, but marginStart flips. 

https://reactnative.dev/blog/2016/08/19/right-to-left-support-for-react-native-apps#overview-changes-in-rn-for-rtl-support

Sounds like a RNW bug though if true

Reply:
....oh
https://github.com/microsoft/react-native-windows/issues/4645#issuecomment-616890884